yellowbrick/text/postag.py

```diff
@@ -20,125 +20,371 @@
 ##########################################################################
 
 from yellowbrick.text.base import TextVisualizer
+from yellowbrick.style.colors import resolve_colors
+
+
+##########################################################################
+# Part-of-speech tag punctuation and labels
+##########################################################################
+
+PUNCT_LIST = [".", ":", ",", "``", "''", "(", ")", "#", "$"]
```

Comment:
NLTK's implementation of Penn Treebank automatically converts all sentence closers, including `!` and `?`, to periods. For now, these tags only get used for Penn Treebank (SpaCy turns everything to `PUNCT`), though I imagined that this may change as other tagsets got included. To ensure no one gets confused about this, I've added a note and changed the var name. Also changed `list` to `frozenset`